packages/core/src/App/Containers/Layout/header/trading-hub-header.jsx

```diff
@@ -186,13 +260,32 @@ const TradingHubHeader = ({
                                 <Icon icon='IcUserOutline' size={20} />
                             </BinaryLink>
                         </Popover>
+                        {cashier_routes && (
+                            <div className='trading-hub-header__menu-right--items--account-toggle'>
+                                <AccountInfo
+                                    acc_switcher_disabled_message={acc_switcher_disabled_message}
+                                    account_type={account_type}
+                                    balance={
+                                        typeof balance === 'undefined' ? balance : formatMoney(currency, balance, true)
```

Comment:
Is it right check? We pass balance only if it's undefined??? Another case we pass string, correct?

Reply:
My bad I forgot to update this aswell 🙏🏻